### PR TITLE
(WIP) (MAINT) Backport Jenkins struct to 8.2

### PR DESCRIPTION
**Pull request by jcoconnor**
Apply the JJB/Jenkins Job structure changes and eliminate PUPPET_NFS
(use aio puppet configuration method)

### CODE REVIEW

File: templates/debian-8.2/i386.vmware.base.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
   "variables":
     {
       "template_name": "debian-8.2-i386",
-      "template_os": "debian8",
+      "template_os": "debian7",
```
Comment (shrug) [inline templates/debian-8.2/i386.vmware.base.json]: should this be debian7?

File: templates/debian-8.2/x86_64.vmware.base.json [hunk 2]
```diff
@@ -3,7 +3,8 @@
   "variables":
     {
       "template_name": "debian-8.2-x86_64",
-      "template_os": "debian8-64",
+      "template_os": "debian7-64",
```
Comment (shrug) [inline templates/debian-8.2/x86_64.vmware.base.json]: Should this be debian7?

File: templates/debian-8.2/i386.vmware.base.json (hunk 1 above)
Comment (jcoconnor) [inline templates/debian-8.2/i386.vmware.base.json]: @shrug Yes - it should be, but the version of ```vmware-workstation-10.0``` on the build machines doesn't recognise higher than ```debian7```

File: templates/debian-8.2/x86_64.vmware.base.json (hunk 2 above)
Comment (jcoconnor) [inline templates/debian-8.2/x86_64.vmware.base.json]: See above

File: templates/debian-8.2/i386.vmware.base.json (hunk 1 above)
Comment (shrug) [inline templates/debian-8.2/i386.vmware.base.json]: 👍

**Comment by shrug** [comment]
is this still "don't merge"?

**Comment by jcoconnor** [comment]
@shrug I'm closing this as have raised ticket and new PR #194 for this work (turned out to be a little more extensive than a simple MAINT task.

**Pull request closed without merging**